Source/relay/TourGuide/Quests/Level 6.ash

```diff
@@ -23,13 +23,20 @@ void QLevel6Init()
 float QLevel6TurnsToCompleteArea(location place)
 {
     //FIXME not sure how accurate these calculations are.
-    int turns_spent_in_zone = turnsAttemptedInLocation(place); //not always accurate
     int ncs_found = noncombatTurnsAttemptedInLocation(place);
 
+
+    // get_property("lastFriarsXNC").to_int() will return 0 until the first NC is hit, then it will return the turns spent in the zone prior to hitting the NC,
```

Comment:
i think i get this but can we add an example? just to add some clarity? suggested ex, and please fix this if i misread the comment, as i think this describes what you described:

```
// For example, if you spend 3 turns in the neck, hit an NC, then hit another NC after 2 turns the pref will manifest as:
//     TURN 1: lastFriarsNeckNC = 0  
//     TURN 2: lastFriarsNeckNC = 0
//     TURN 3: lastFriarsNeckNC = 0
//     TURN 4: lastFriarsNeckNC = 0 <= hits NC
//     TURN 5: lastFriarsNeckNC = 1
//     TURN 6: lastFriarsNeckNC = 2
//     TURN 7: lastFriarsNeckNC = 3 <= hits NC
```

Reply:
You got it, for sure. I remember this being a little weird to puzzle out so making it as clear as possible to anyone else that looks at the code is definitely a good idea.